sound/soc/sof/pcm.c

```diff
@@ -325,6 +325,42 @@ static int sof_pcm_trigger(struct snd_pcm_substream *substream, int cmd)
 	/* send IPC to the DSP */
 	ret = sof_ipc_tx_message(sdev->ipc, stream.hdr.cmd, &stream,
 				 sizeof(stream), &reply, sizeof(reply));
+	if (ret < 0)
+		return ret;
+
+	/* do we need to inform runtime PM we can enter D3 ? */
+	permit_d0i3 = snd_sof_pipeline_permit_d0i3(sdev,
+						   spcm->swidget->pipeline_id);
```

Comment:
Is this correct? I thought the information that the dmic dailink is configured with .ignore_suspend was enough for DAPM/ALSA. @ranj063 can you comment?

Reply:
@plbossart @lgirdwood yes ignore_suspend is already defined for the DMIC dai link that is used for WoV in the machine driver. This is already used to determine if the stream should be suspended when the devices enters S0ix.